cognite/client/_api/files.py

```diff
@@ -72,6 +72,54 @@ def __iter__(self) -> Generator[FileMetadata, None, None]:
         """
         return self.__call__()
 
+    def create(
+        self,
+        name: str,
+        external_id: str = None,
+        source: str = None,
+        metadata: Dict[str, Any] = None,
+        asset_ids: List[int] = None,
+        overwrite: bool = False,
```

Comment:
You can just take a FileMetadata object as parameter here, that way you won't have to worry about keeping this signature in sync with the auto-generated FileMetadata constructor signature. You will still need to include the overwrite parameter of course.

You are also missing the return type hint here which should be Tuple[FileMetadata, str].